### PR TITLE
docs(scss): add migration guide for scss mixin and media queries

### PR DESCRIPTION
**Pull request by paul-vd**
### What?
This adds the manual migration guide for scss mixin and media queries

### CODE REVIEW

File: docs/migration/manual-migration.mdx [hunk 1]
```diff
@@ -13,69 +13,103 @@ parts, or debug issues in the migration CLI output.
...
+
+After running the codemods on your project, you can easily find these
+deprecations by searching `TODO (Codemod generated)` in your project.
+
+:::
+
+#### `grid-breakpoints` Variable
 
-If you make use of the
+The `grid-breakpoints` variable has been deprecated, If you make use of the
```
Comment (Kagaminara) [inline docs/migration/manual-migration.mdx:66]:
```suggestion
The `grid-breakpoints` variable has been deprecated, if you make use of the
```
**alternative:**
```suggestion
The `grid-breakpoints` variable has been deprecated. If you make use of the
```

File: docs/migration/manual-migration.mdx [hunk 2]
```diff
@@ -13,69 +13,103 @@ parts, or debug issues in the migration CLI output.
...
+[`grid-breakpoints`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/01f9c95003f7d4204acd12264c3375464e6c3a12/theme-chocolatine/web/theme/components/atoms/Breakpoints/_breakpoints.scss#L4-12)
+variable, you will need to manually implement this variable in your project.
+
+#### `breakpoint-min` Function
+
+The `breakpoint-min` function has been deprecated, If you make use of the
```
Comment (Kagaminara) [inline docs/migration/manual-migration.mdx:72]:
```suggestion
The `breakpoint-min` function has been deprecated, if you make use of the
```
**alternative:**
```suggestion
The `breakpoint-min` function has been deprecated. If you make use of the
```

File: docs/migration/manual-migration.mdx [hunk 3]
```diff
@@ -13,69 +13,103 @@ parts, or debug issues in the migration CLI output.
...
+[`media-breakpoint-up`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/01f9c95003f7d4204acd12264c3375464e6c3a12/theme-chocolatine/web/theme/components/atoms/Breakpoints/_breakpoints.scss#L14-21)
+function, you will need to manually implement this function in your project.
+
+#### `media-breakpoint-up` Mixin
+
+The `media-breakpoint-up` mixin has been deprecated, If you make use of the
```
Comment (Kagaminara) [inline docs/migration/manual-migration.mdx:78]:
```suggestion
The `media-breakpoint-up` mixin has been deprecated, if you make use of the
```
**alternative:**
```suggestion
The `media-breakpoint-up` mixin has been deprecated. If you make use of the
```

**Review by Kagaminara — APPROVED** [review]
Only nit comments, lgtm otherwise.
No need to request for another review on my end.

**Comment by real34** [comment]
Preview: https://deploy-preview-727--heuristic-almeida-1a1f35.netlify.app/docs/remixed/migration/manual-migration#sass-migration

File: docs/migration/manual-migration.mdx [hunk 4]
```diff
@@ -13,69 +13,103 @@ parts, or debug issues in the migration CLI output.
...
+allows for a more modular approach to theming and a better developer experience.
 However, there are still some manual steps that need to be taken to complete the
 migration.
 
-### Media Mixin
+### Breakpoints Deprecations
+
+:::info
+
+After running the codemods on your project, you can easily find these
```
Comment (real34) [inline docs/migration/manual-migration.mdx:59]:
```suggestion
After running the codemods on your project, you can find these
```

File: docs/migration/manual-migration.mdx [hunk 5]
```diff
@@ -13,69 +13,103 @@ parts, or debug issues in the migration CLI output.
...
 
-If you make use of the
+The `grid-breakpoints` variable has been deprecated, If you make use of the
+[`grid-breakpoints`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/01f9c95003f7d4204acd12264c3375464e6c3a12/theme-chocolatine/web/theme/components/atoms/Breakpoints/_breakpoints.scss#L4-12)
+variable, you will need to manually implement this variable in your project.
+
+#### `breakpoint-min` Function
+
+The `breakpoint-min` function has been deprecated, If you make use of the
+[`media-breakpoint-up`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/01f9c95003f7d4204acd12264c3375464e6c3a12/theme-chocolatine/web/theme/components/atoms/Breakpoints/_breakpoints.scss#L14-21)
```
Comment (real34) [inline docs/migration/manual-migration.mdx:73]: **issue:** incorrect link and name

**suggestion:** use `breakpoint-min`'s correct one

**Review by real34 — CHANGES_REQUESTED** [review]

File: docs/migration/manual-migration.mdx (hunk 5 above)
Comment (paul-vd) [reply docs/migration/manual-migration.mdx:73]: Hmm not sure I follow it is breakpoint-min?

image

Comment (real34) [reply docs/migration/manual-migration.mdx]: The link seems similar to the other section: 


I meant that it very likely was a copy/paste error.

File: docs/migration/manual-migration.mdx [hunk 6]
```diff
@@ -13,69 +13,95 @@ parts, or debug issues in the migration CLI output.
...
-### Media Mixin
+### Breakpoints Removal
 
-If you make use of the
-[`media-breakpoint-up`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/01f9c95003f7d4204acd12264c3375464e6c3a12/theme-chocolatine/web/theme/components/atoms/Breakpoints/_breakpoints.scss#L23-34)
-mixin, you will need to directly import the
-[\_breakpoints.scss](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/01f9c95003f7d4204acd12264c3375464e6c3a12/theme-chocolatine/web/theme/components/atoms/Breakpoints/_breakpoints.scss)
-file at the location where it is used.
+The following scss helpers have been removed. If you were using them, you have
+to manually add them back to your project or update implementation
```
Comment (real34) [inline docs/migration/manual-migration.mdx:58]: > or update implementation

How would an integrator know what they should do? The codemod comment states "Please follow our manual migration guide: https://developers.front-commerce.com/docs/remixed/migration/manual-migration#breakpoints-removal" but the migration guide doesn't bring any value IMO.

**suggestion:** document how the implementation could be updated to not use these removed variables (i.e: do like in our core)

**Review by mGGk-fr — APPROVED** [review]

File: docs/migration/manual-migration.mdx (hunk 6 above)
Comment (real34) [reply docs/migration/manual-migration.mdx:58]: I added another section with a more recommended approach, to replace userland code with simpler and direct media queries in d2a22b3c1e7d4631dea4223031ade5045a56246c

**Review by real34 — APPROVED** [review]
I directly wrote what I meant in previous comments in d2a22b3c1e7d4631dea4223031ade5045a56246c.

As discussed, I'll merge this PR so we can move on.

**Pull request merged**